warehouse/ingest-core/src/main/java/datawave/ingest/metadata/EventMetadata.java

```diff
@@ -310,15 +310,26 @@ protected void addTokenizedContent(IngestHelperInterface helper, RawRecordContai
         if (helper instanceof AbstractContentIngestHelper) {
             String tokenDesignator = Objects.toString(((AbstractContentIngestHelper) helper).getTokenFieldNameDesignator(), "");
             AbstractContentIngestHelper h = (AbstractContentIngestHelper) helper;
+            boolean contentIndexed;
             for (String field : fields.keySet()) {
+                contentIndexed = false;
                 if (h.isContentIndexField(field)) {
                     updateForIndexedField(helper, event, fields, countDelta, loadDate, tokenDesignator, field);
                     termFrequencyFieldsInfo.createOrUpdate(field + tokenDesignator, event.getDataType().outputName(),
                                     MetadataWithMostRecentDate.IGNORED_NORMALIZER_CLASS, event.getDate());
+                    contentIndexed = true;
                 }
                 
                 if (h.isReverseContentIndexField(field)) {
                     updateForReverseIndexedField(helper, event, fields, countDelta, loadDate, tokenDesignator, field);
+                    contentIndexed = true;
+                }
+                
+                if (contentIndexed) {
+                    log.debug(field + tokenDesignator + " has a data type");
```

Comment:
This should either be wrapped in log.isDebugEnabled, or use a parameterized log to avoid wasting time on doing the string concatenation when not logging.

Reply:
Modified in the next commit

Reply:
Applied parameterized logging for all the debug logging.